core/src/main/scala/fordeckmacia/Faction.scala

```diff
@@ -18,8 +18,21 @@ object Faction:
   case object Shurima         extends Faction { val id = "SH"; val int = 7  }
   case object MountTargon     extends Faction { val id = "MT"; val int = 9  }
   case object BandleCity      extends Faction { val id = "BC"; val int = 10 }
+  case object Runeterra       extends Faction { val id = "RU"; val int = 11 }
```

Comment:
Possible heads up, think Runeterra's ID is `12`
https://github.com/RiotGames/LoRDeckCodes/commit/680c25d6a9ec68095becf4c4ebbda6760ba358ae#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R76

Reply:
I saw that! Thanks for the heads up though 🙏 I'll make sure to fix it before I merge it